src/ladder.ts

```diff
@@ -364,7 +364,11 @@ export class GlickoPlayer {
 		// Follow along the steps using: http://www.glicko.net/glicko/glicko.pdf
 
 		if (m.length === 0) {
-			const RD = Math.sqrt((this.rd * this.rd) + (this.c * this.c));
+			const RD = Math.min(this.RDmax, 
+				Math.max(this.RDmin, 
+					Math.sqrt((this.rd * this.rd) + (this.c * this.c))
+				)
+			);
```

Comment:
However, I'm not sure about this one; [the original PHP](https://github.com/smogon/pokemon-showdown-client/blob/399aee9f74e2f4bebf3fc40cf5eb5b416942e764/lib/ntbb-ladder.lib.php#L67-L70) doesn't force the value of `RD` into an interval like this.

The Glicko paper itself doesn't seem to set a lower bound on RD, though it does specify an upper one.
In any case, I'm concerned about changing ladder behavior beyond just fixing regressions from the new loginserver. Is there somewhere in the PHP where `RD` is clamped like this that I'm not seeing?